src/DynamoCore/Engine/NodeToCode.cs

```diff
@@ -126,338 +212,338 @@ public int ActionCount()
         }
     }
 
```

Comment:
The following section in this file seems to show changes due to some code movements, it's hard to identify what has changed. I only looked at `NamingProvider` above. Is there anything I miss in this file?